src/current/_includes/v23.1/ui/insights.md

```diff
@@ -89,22 +95,29 @@ The rows in this page are populated from the [`crdb_internal.cluster_execution_i
 {% endif %}
 {{site.data.alerts.end}}
 
-To view [details of the execution](#statement-execution-details), click an execution ID in the **Statement Execution ID** column.
-
-- **Statement Execution ID**: The execution ID of the latest execution with the statement fingerprint.
-- **Statement Fingerprint ID**: The statement fingerprint ID of the latest statement execution.
-- **Statement Execution**: The [statement fingerprint]({{ link_prefix }}ui-statements-page.html#sql-statement-fingerprints) of the latest statement execution.
-- **Insights**: The [insight type](#workload-insight-types) for the statement execution.
-- **Start Time (UTC)**: The timestamp when the statement execution started.
-- **Elapsed Time**: The time that elapsed to complete the statement execution.
-- **User Name**: The name of the user that invoked the statement execution.
-- **Application Name**: The name specified by the [`application_name`]({{ link_prefix }}show-vars.html#supported-variables) session setting.
-- **Rows Processed**: The total number of rows read and written.
-- **Retries**: The number of times the statement execution was [retried]({{ link_prefix }}transactions.html#automatic-retries).
-- **Contention Time**: The amount of time the statement execution spent waiting in [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention).
-- **Full Scan**: Whether the execution performed a full scan of the table.
-- **Transaction Execution ID**: The ID of the transaction execution for the statement execution.
-- **Transaction Fingerprint ID**: The ID of the transaction fingerprint for the statement execution.
+Click **Columns** to select the columns to display in the table.
+
+The statement insights table has the following columns available:
```

Comment:
nit: Bold/capitalize the table name (assuming that's the name of the table in the UI)

Reply:
I am not capitalizing and bolding "statement insights" because It is not the title of the table, but the contents of the table:
<img width="522" alt="Screenshot 2023-06-12 at 10 12 42 AM" src="https://github.com/cockroachdb/docs/assets/58752716/6740be31-f47d-4e77-bd2e-5727758fb613">